lib/cm_clickpost.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# require_relative "cm_clickpost/version"
```

Comment:
Use single quotes bro. Check all the other lines.

Reply:
We don't need it actually. I'm looping through the files and requiring it. Will remove it